app/controllers/org_admin/templates_controller.rb

```diff
@@ -104,12 +94,8 @@ def update
       @template = Template.find(params[:id])
       authorize @template   # NOTE if non-authorized error is raised, it performs a redirect to root_path and no JSON output is generated
 
-      current = Template.current(@template.family_id)
-
       # Only allow the current version to be updated
-      if current != @template
-        render(status: :forbidden, json: { msg: _("You can not edit a historical version of this #{template_type(@template)}.")})
-      else
+      if @template.latest?
```

Comment:
you have to add here the logic to generate a new template version if the latest is published

Reply:
am holding off on that for the final ticket (although I don't think it needs to be a concern as the name of that ticket suggests). We should only version the template when 'versionable' fields have changed. I want us to clarify what those things are. For example if the description changes, or the visibility changes it probably doesn't need a new version so we'd want to see if the title and other versioanble fields are dirty.